cincinnati/src/plugins/internal/channel_filter.rs

```diff
@@ -56,14 +57,17 @@ impl InternalPlugin for ChannelFilterPlugin {
             self.key_suffix.to_owned(),
         ))
         .and_then(|(internal_io, key_prefix, key_suffix)| {
-            let channel = get_multiple_values!(internal_io.parameters, "channel")?.clone();
+            let channel = match get_multiple_values!(internal_io.parameters, "channel"){
+                Ok(c) => c.clone(),
+                Err(e) => return Err(GraphError::MissingParams(vec!(e.to_string()))),
```

Comment:
This should be `vec!["channel".to_string()]`.

Reply:
I was trying to avoid hardcoding "channel" in yet another place, but sure, I'll change that

Reply:
Sorry, let me rephrase for clarity: the vector should contain only the keyname of the missing parameter, not a full human-friendly error string.